### PR TITLE
Basepath 

### PR DESCRIPTION
**Pull request by simonard**
added so when switching between local and ec2 only need to change one line of code

### CODE REVIEW

File: src/apiBasePath.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/**
+ * The api base path exported here is used to connect to the back-end API
+ * EC2 path: http://ec2-18-232-171-89.compute-1.amazonaws.com:8081/
+ * LocalHost path: http://localhost:8081/
+ */
+
+const apiBasePath = 'http://localhost:8081/';
```
Comment (smgruver) [inline src/apiBasePath.js]: Leave out the ending '/' to be included if necessary in the string literal of requests.

**Review by smgruver — CHANGES_REQUESTED** [review]
Just remove the ending slash from the apiBasePath export and re-add them to the HTtp request URLs

File: src/Components/login/Login.jsx [hunk 2]
```diff
@@ -81,7 +82,7 @@ export default function Login(props) {
       redirect: "follow",
     };
 
-    fetch("http://ec2-18-232-171-89.compute-1.amazonaws.com:8081/authenticate", requestOptions)
+    fetch(apiBasePath + "authenticate", requestOptions)
```
Comment (AnthonyNGarcia) [inline src/Components/login/Login.jsx]: Missed slash here for "/authenticate"

**Review by smgruver — CHANGES_REQUESTED** [review]
Remove the ending '/' from the commented URLs in apiBasePath & re-add the '/' for the /authenticate in login.jsx

**Review by smgruver — APPROVED** [review]

**Review by AnthonyNGarcia — APPROVED** [review]

**Pull request merged**